libreoffice-core/sw/source/core/doc/DocumentRedlineManager.cxx

```diff
@@ -1274,6 +1274,13 @@ DocumentRedlineManager::AppendRedline(SwRangeRedline* pNewRedl, bool const bCall
         return AppendResult::IGNORED;
     }
 
+    // MACRO-1266 - Do not create a delete redline for a comment
```

Comment:
I like this idea a lot

Reply:
Good call, also lets us see at a glance if something was introduce by us vs. upstream. Will be handy for merge conflicts going forward.